readme/email_to_note.md

```diff
@@ -0,0 +1,27 @@
+# Email to note
+
+You can save your emails in Joplin Cloud by forwarding your emails to your Inbox Email address. The content of the email and its attachments will be saved in a note with the name of the email inside a notebook called Inbox. 
+
+All emails forwarded to your Inbox Email will use your Joplin Storage capacity.
+
+This feature is available for Pro and Teams members.
+
+## How to get your Inbox Email
+
+### Desktop
+
+To copy your Inbox Email you will need to navigate to the Options screen and then to the Joplin Cloud tab. You will need to have your synchronization set to Joplin Cloud.
+
+<img src="https://raw.githubusercontent.com/pedr/joplin/email-to-note-docs/Assets/WebsiteAssets/images/email_to_note/desktop.png" width="80%"/>
```

Comment:
The URL should be the one where the content will be once we merge the pull request. So it should be `https://raw.githubusercontent.com/laurent22/joplin/dev/Assets/WebsiteAssets/images/email_to_note/desktop.png` and same for the other links